doc/manual/en/06-backup.md

```diff
@@ -125,10 +125,97 @@ Response:
+* Copy all the WAL segments after and including the WAL segment in which start lsn is present
+* Generate manifest file over the incremental backup data directory
+
+### Dependencies
+
+For PostgreSQL version 13-16, we rely on `pgmoneta` native block-level incremental solutions for backups. To facilitate this solution `pgmoneta` highly depends on a custom extension [pgmoneta_ext](github.com/pgmoneta/pgmoneta_ext) and postgres's system administration functions. Following are the list of admin functions `pgmoneta` depends on:
+
+| Function                    | System administration function | Minimum Privilege | Parameters | Description                                            |
+|-----------------------------|---|--------|------------|--------------------------------------------------------|
+| `pgmoneta_server_backup_start`    |   pg_start_backup/pg_backup_start |    EXECUTE    | label | Returns a row with the backup start lsn |
+| `pgmoneta_server_backup_stop`    |   pg_stop_backup/pg_backup_stop |    EXECUTE    | None | Returns a row with two columns - backup stop lsn and the backup label file contents  |
+| `pgmoneta_server_read_binary`    |   pg_read_binary_file |    pg_read_server_files & EXECUTE    | (offset, length, path/to/file) | Returns the contents of the file provided from the server of particular length at a particular offset |
+| `pgmoneta_server_file_stat`    |   pg_stat_file |    pg_read_server_files & EXECUTE    | path/to/file | Returns the metadata of the file provided from the server like file size, modification time etc |
+
+For complete information on the server api refer [this](https://github.com/pgmoneta/pgmoneta/blob/main/doc/manual/en/80-server-api.md)
+
+The extension functions on which the native solution depends are:
+
+- `pgmoneta_ext_get_file('<path/to/file>')`
+- `pgmoneta_ext_get_files('<path/to/dir>')`
+
+You can read more about these functions and their required privileges over [here](https://github.com/pgmoneta/pgmoneta_ext/blob/main/doc/manual/en/04-functions.md)
+
+### Setup
+
+Let's assume we want to make setup for PostgreSQL 14. Make sure PostgreSQL 14 is installed on your system. Since our solution depends on `pgmoneta_ext`, build and install the extension for version 14 using the following commands:
+
+```
+git clone https://github.com/pgmoneta/pgmoneta_ext
+cd pgmoneta_ext
+mkdir build
+cd build
+cmake ..
+make
+sudo make install
+```
+
+If multiple postgres versions are present modify the `PATH` variable such that binary path of version 14 appears first. For example if the current `echo $PATH` looks like this:
+
+```
+PATH=...:/usr/lib/postgresql/17/bin:...:/usr/lib/postgresql/14/bin:...
+```
+
+make it and then build and install
+
+```
+PATH=...:/usr/lib/postgresql/14/bin:...:/usr/lib/postgresql/17/bin:...
+```
+
+Lastly, inside the postgres cluster (version 14) perform following commands (assuming a replication user is present):
+
+```
+-- create extension
```

Comment:
do this as a shell script such that people can do `./incr-permissions.sh myrepl`